packaging/rpm/postinst

```diff
@@ -1,19 +1,19 @@
 #!/bin/sh
-set -e
+# set -e
```

Comment:
nit: Can you add a comment as to why we are ignoring errors and not failing?

Reply:
Done